README.md

```diff
@@ -1,65 +1,72 @@
-# React and Redux
+# React - Math Magicians (Calculator)
 
-If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Set-up GitHub Actions
+# Math Magicians
 
-This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.
+> This application use React framework to show a calculator.
```

Comment:
- [ ] Your live demo link is pointing to an other project. Please double check
![Screenshot from 2022-05-25 11-02-52](https://user-images.githubusercontent.com/65860678/170225284-ff8329ab-8e3b-40e1-99d8-4ad802544158.png)